src/types/options/index.ts

```diff
@@ -36,36 +36,62 @@ export type KnownOptionsPayloadFromType<T extends string> = Extract<SlackOptions
  */
 export interface BlockSuggestion extends StringIndexed {
   type: 'block_suggestion';
-  block_id: string;
   action_id: string;
-  value: string;
-
   api_app_id: string;
+  block_id: string;
+  channel?: {
+    id: string;
+    name: string;
+  };
+  is_enterprise_install?: boolean;
+  enterprise?: {
+    id: string;
+    name: string;
+  } | null;
```

Comment:
I'm curious, why do we need | null if the field is optional?

Reply:
great callout! This was actually a result of meshing together the two schemas 😭 in the Deno schema, the `| null` is there and it's not an optional field, so I think I accidentally left that in as I was going through to make sure it aligned with both schemas. I'll remove the `| null` from this!

Reply:
Oh gotcha! No big deal, but yeah we can safely remove that :)